frontend/src/config/store.ts

```diff
@@ -1,17 +1,25 @@
-import { configureStore } from "@reduxjs/toolkit";
-import { combineReducers } from "redux";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
What's the reason you don't want to type-check this file? I'm unsure as to how the `MigrationManifest` works, but I see removing this line does cause compilation to fail.

Reply:
Oh yea, didn't want to deal with trying to create types with the old redux state using the newer redux state types as you need to omit/transform the type if there's a major change with the structure. Would not be worth spending the time to do so as we will be moving the persistance to the BE soon 🤞